src/home/org.freedesktop.home1.policy

```diff
@@ -49,6 +49,16 @@
                 </defaults>
         </action>
 
+        <action id="org.freedesktop.home1.update-own-home">
```

Comment:
dunno, but wouldn't it make more sense to put "-own" in as a suffix, because it is a specialization? dunno, was just wondering, i.e. `org.freedesktop.home1.update-home-own`

Reply:
`update-home-own` just sounds strange grammatically, but I see your point

Thoughts on `update-home-safe`?

Reply:
Or `update-home-limited`? `update-home-by-owner`?

Reply:
maybe "-self"? 

"-by-owner" sounds ok to me.

"-limited" and "-safe" sounds a bit too unspecific.